apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -93,8 +93,8 @@ const AppearanceView = () => {
       <Meta title={t("appearance")} description={t("appearance_description")} />
       <div className="mb-6 flex items-center text-sm">
         <div>
-          <p className="font-semibold">{t("theme")}</p>
-          <p className="text-gray-600">{t("theme_applies_note")}</p>
+          <p className="text-default font-semibold">{t("theme")}</p>
+          <p className="text-default">{t("theme_applies_note")}</p>
```

Comment:
<img width="984" alt="Screenshot 2023-04-03 at 4 48 10 PM" src="https://user-images.githubusercontent.com/53316345/229494497-09efc796-95ae-428f-9fd3-3659e82808ac.png">

Reply:
Added to my list

Reply:
this doesn't look right

Reply:
Will fix <3